src/vs/vscode.d.ts

```diff
@@ -1987,6 +1987,13 @@ declare module 'vscode' {
 		 */
 		readonly diagnostics: Diagnostic[];
 
+		/**
+		 * The current selection in the editor.
+		 *
+		 * Undefined if there is no current selection.
```

Comment:
`undefined` instead Undefined

Reply:
We should explain when there is no selection, e.g. there is always a selection with an editor but we should point out that providers aren't always called without the document being shown inside an editor